docs/general/adapters.md

```diff
@@ -14,47 +43,89 @@ Doesn't follow any specific convention.
 
 ### JSON
 
-It also generates a json response but always with a root key. The root key **can't be overridden**, and will be automatically defined accordingly to the objects being serialized.
+The response document always with a root key.
+
+The root key **can't be overridden**, and will be derived from the resource being serialized.
+
 Doesn't follow any specific convention.
 
 ### JSON API
 
 This adapter follows **version 1.0** of the format specified in
```

Comment:
We may want to consider the implementation of the JSON API objects external to the JSON API adapter, as discussed in https://github.com/rails-api/active_model_serializers/pull/1004#discussion_r46783510 (and relevant to the above comment)